src/integrationTest/groovy/wooga/gradle/release/ReleasePluginIntegrationSpec.groovy

```diff
@@ -440,6 +440,7 @@ class ReleasePluginIntegrationSpec extends IntegrationSpec {
         "final"   | false
         "release" | false
         "rc"      | true
+        "pre"     | true
```

Comment:
I thought we call it `prefligh` everywhere

Reply:
I am adding it onto atlas-version.